services/account/src/models/account.definition.ts

```diff
@@ -0,0 +1,34 @@
+export const accountDefinition = {
```

Comment:
I think we can decorate the model by 

```ts
@model
Class Account {
  @property('id') id:string,
  // ...etc
}
```

If any controller method has argument with type `Account`, then the OpenAPI `schema` for `Account` will be auto-generated.